app/views/mortgage_calculator/affordabilities/_form_step1.html.erb

```diff
@@ -44,22 +44,52 @@
             </span>
           <% end %>
         </div>
+      </div>
+
+      <div class="affcalc__row">
+        <div class="affcalc__col--field">
+          <%= person.form_row :monthly_net_income do %>
+            <%= person.errors_for :monthly_net_income %>
+            <%= person.label :monthly_net_income, "id" => "label_monthly_net_income_0" do %>
+              <%= t("affordability.activemodel.attributes.mortgage_calculator/person.monthly_net_income")%> * 
+              <span class="affcalc__row affcalc__sub-label"><%= t("affordability.activemodel.attributes.mortgage_calculator/person.monthly_net_sub")%></span>
+              <span class="visually-hidden"><%= t("affordability.per_month") %>, <%= t("affordability.required") %></span>
+            <% end %>
 
-        <div class="affcalc__col--tooltip">
-          <%= field_tooltip text: t("affordability.tooltips.mortgage_calculator/person.annual_income_0_html"),
-                html: {
-                  id: 'tip_annual_income_0'
-                }
-            %>
+            <span class="form__input-container">
+              <span class="form__input-label" id="label_monthly_net_income_0a">&pound;</span>
+              <%= person.text_field :monthly_net_income,
+                                    "class" => "form-control form__input",
+                                    "ng-model" => "affordability.earnings.person0.net_pay",
+                                    "ng-initial" => "",
+                                    "currency" => '',
+                                    "placeholder" => "0",
+                                    "aria-describedby" => "tip_monthly_net_income_0",
+                                    "aria-labelledby" => "label_monthly_net_income_0 label_monthly_net_income_0a label_monthly_net_income_0b",
+                                    "required" => "required",
+                                    "aria-required" => "true",
+                                    "value" => person.object.monthly_net_income_formatted,
+                                    "pattern" => '\d*' %>
+              <span class="form__input-label" id="label_monthly_net_income_0b"><%= t("affordability.per_month") %></span>
```

Comment:
Can we use single-quotes on the ruby erb part, please? From this line until the beginning.